docs/beta/plugins/discord.md

```diff
@@ -1,5 +1,248 @@
 ---
 title: Discord
+description: Setting up the Discord plugin.
 ---
 
-TODO
+The Discord plugin assigns Cred to contributors participating in Discord chats; for example, by posting and reacting to messages. 
+
+
+
+The plugin does this using a special bot that scans a Discord server and captures all messages and emoji reactions. The data is temporarily cached, but the
+source of truth for the Discord plugin is always the live
+Discord server. That means that if content is deleted on Discord, it will
+also disappear from the Discord plugin (after a cache refresh) {SB: copied from Discourse plugin, true for Discord also, right?}.
+
+
+For instructions on setting up the bot and configuring the plugin, see the [Configuration](#configuration) section below. 
+
+
+## Cred flow
+
+Cred is minted on Discord reactions. It then flows to connected nodes.
+
+TO DO: add example with diagram....The below diagram illustrates a typical pattern of Cred flow in Discord....
+
+
+
+The full set of node and edge types used by the Discord plugin are defined below.
+
+### Nodes
+
+The Discord plugin creates the following types of nodes in the contribution graph:
+
+- **User**:
+
+A Discord user account, e.g. [@decentralion {SB: correct link?}](). User accounts do not mint Cred, so setting a node weight would have no effect. Using the identity plugin, it's possible to "collapse" user nodes with other identity nodes into a single, canonical identity. For example, if a contributor had a Discord user account and a GitHub account, then the identity plugin can collapse those identities together.
+
+Users are connected to messages they author, to messages they react to, and to messages that mention them {SB: is this true?}.
+
+- **Guild Member**:
+
+A Discord user account, e.g. [@decentralion {SB: correct?}](), that has special powers. E.g. their reactions mint more Cred {SB: totally making this up here. Is this part of beta launch, or just a hidden alpha feature for MetaGame?}
```

Comment:
I would just take this out since it's not implemented right now and is not clear how exactly we will make it work. Even then, it wont be an actual node type.

Reply:
Removed

Reply:
Removed.